packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1043,15 +1043,48 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
   // The expiration date is calculated to be at midnight UTC, a specific number of days from now.
   mintCapacityCreditsNFT = async ({
     requestsPerDay,
+    requestsPerSecond,
+    requestsPerKilosecond,
     daysUntilUTCMidnightExpiration,
-  }: MintCapacityCreditsPerDayContext): Promise<MintCapacityCreditsRes> => {
+  }: MintCapacityCreditsContext): Promise<MintCapacityCreditsRes> => {
     this.log('Minting Capacity Credits NFT...');
 
-    // -- in the context of "request per day"
-    const requestsPerSecond = convertRequestsPerDayToPerSecond(requestsPerDay);
-    const requestsPerKilosecond = Math.round(
-      calculateRequestsPerKilosecond(requestsPerSecond)
-    );
+    // Validate input: at least one of the request parameters must be provided
+    if (!requestsPerDay && !requestsPerSecond && !requestsPerKilosecond) {
+      throw new Error(
+        'At least one of requestsPerDay, requestsPerSecond, or requestsPerKilosecond is required'
+      );
+    }
```

Comment:
Does it make sense to have one set to 0? If that is valid, this check will throw. Also if they pass a negative value, this check succeeds

Reply:
Yea it doesn't - but this will also be caught by the smart contract and throw the correct error message (eg. minimum x is required)